experimental/ubuntu/ubuntu18.04-minimal/ubuntu18.04-minimal

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:18.04
+
+RUN apt-get update \
+  && apt-get install --yes \
+    openjdk-8-jdk \
+    iproute2 \
+    bash \
+    sudo \
+    coreutils \
+    procps \
+    strace \
```

Comment:
strace should be installed for users who are trying to set withStrace on their clusters